src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -597,6 +587,7 @@ void AutomatedRssDownloader::handleFeedCheckStateChange(QListWidgetItem *feed_it
     updateMatchingArticles();
 }
 
+// FIXME: This should return early if any of the filters are invalid
```

Comment:
That's not entirely true, which is why I added the comment. I misunderstood how the episode filter worked originally and had entered an invalid one in the form "3x5-12;8x10-", which is invalid, but still gets run since it matches the simpler pattern used by the matching article search.